sdk/typespec/typespec_client_core/src/async_runtime/standard_runtime.rs

```diff
@@ -150,7 +150,7 @@ impl AsyncRuntime for StdRuntime {
     /// Uses a simple thread based implementation for sleep. A more efficient
     /// implementation is available by using the `tokio` crate feature.
     #[cfg_attr(target_arch = "wasm32", allow(unused_variables))]
```

Comment:
Seems this line is no longer needed.

Reply:
If we want to do https://github.com/Azure/azure-sdk-for-rust/pull/2770#discussion_r2195904158, this is still needed to allow a user to select the `StdRuntime` even target to wasm32.